docs/types/value-types.rst

```diff
@@ -261,6 +261,11 @@ reverts on failure.
 .. note::
     If ``x`` is a contract address, its code (more specifically: its :ref:`receive-ether-function`, if present, or otherwise its :ref:`fallback-function`, if present) will be executed together with the ``transfer`` call (this is a feature of the EVM and cannot be prevented). If that execution runs out of gas or fails in any way, the Ether transfer will be reverted and the current contract will stop with an exception.
 
+.. warning::
+    ``transfer`` is deprecated and scheduled for removal in the next breaking version (0.9).
+    Use the :ref:`call function <address_call_functions>` with an optionally provided maximum amount of
+    gas (default forwards all remaining gas) and an empty calldata parameter, e.g., ``call{value: amount}("")``.
+
 * ``send``
```

Comment:
This it not something introduced by the PR, but note that these bullet points are incorrectly nested. We should fix that. The text following them should be indented - otherwise it's outside the list.

We could also just convert them into section headers.

Reply:
I indented the paragraphs after the bullet points.